sandbox/src/main/java/org/robolectric/interceptors/AndroidInterceptors.java

```diff
@@ -465,6 +468,40 @@ public MethodHandle getMethodHandle(String methodName, MethodType type)
     }
   }
 
+  /** Intercepts calls to methods in {@link javax.crypto.Cipher} not present in the OpenJDK. */
+  public static class CipherInterceptor extends Interceptor {
```

Comment:
@alwa Can you write a test as your app does to test this behavior?

Reply:
I'm not sure how since I use Robolectric to trigger the problem and I guess that option is not possible for unit tests in the Robolectric project?

Reply:
Done. I have a test in place now. It fails though (as would be expected. Let's continue the conversation in: https://github.com/robolectric/robolectric/issues/8242#issuecomment-1581144744)